community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf

```diff
@@ -15,11 +15,13 @@
 variable "name" {
   description = "Name of the nodeset."
   type        = string
-  default     = "ghpc"
 
   validation {
     condition     = can(regex("^[a-z](?:[a-z0-9]{0,5})$", var.name))
-    error_message = "Nodeset name (var.name) must begin with a letter, be fully alphanumeric and be 6 characters or less. Regexp: '^[a-z](?:[a-z0-9]{0,5})$'."
+    error_message = <<EOT
+Nodeset name (var.name) must begin with a letter, be fully alphanumeric and be 6 characters or less.
+Got "${var.name}" (set to module id by default).
```

Comment:
here I am wondering if you should not do similarly to what we do in slurm cluster name, just pick a subset of the n first letters, remove spaces, dashes, etc....